prow/jobs/test-infra/check_config_files.yaml

```diff
@@ -0,0 +1,17 @@
+presubmits:
+  kyma-project/test-infra:
+  - name: prow/test-infra/check_config
+    trigger: "(?m)^/yamllint"
```

Comment:
Please make the triggering command more explicit (something like `/test check_config`)